gazebo/gui/model/ModelTreeWidget.cc

```diff
@@ -22,6 +22,9 @@
 #include "gazebo/gui/model/ModelPluginInspector.hh"
 #include "gazebo/gui/model/ModelTreeWidget.hh"
 
+#ifdef __APPLE__
```

Comment:
why are we restricting the use of the namespace to Mac and not use it in all platforms?

Reply:
that's a good point; I would have thought this was an issue with the boost version, not the OS

Reply:
I'll remove these `ifdefs`

Reply:
16f1581f00b5c3c32e4a0eea1e83ae87ac22cf58